src/memberBundle.ts

```diff
@@ -0,0 +1,175 @@
+import { getModelForClass } from '@typegoose/typegoose';
+import { addDays } from 'date-fns';
+import {
+  issueMemberCertificate,
+  retrieveVeraDnssecChain,
+  selfIssueOrganisationCertificate,
+  serialiseMemberIdBundle,
+} from '@relaycorp/veraid';
+import type { BaseLogger } from 'pino';
+
+import type { ServiceOptions } from './serviceTypes.js';
+import { MemberPublicKeyModelSchema } from './models/MemberPublicKey.model.js';
+import { MemberModelSchema } from './models/Member.model.js';
+import { Kms } from './utilities/kms/Kms.js';
+import { OrgModelSchema } from './models/Org.model.js';
+import { derDeserialisePublicKey } from './utilities/webcrypto.js';
+import type { Result } from './utilities/result.js';
+
+const CERTIFICATE_EXPIRY_DAYS = 90;
+interface CreateBundleInput {
```

Comment:
```suggestion
interface BundleCreationInput {
```

Following our code conventions.